htdocs/web_portal/views/user/view_user.php

```diff
@@ -122,10 +122,12 @@
                     <tr class="site_table_row_2">
                         <td class="site_table">Home Site</td>
                         <td class="site_table">
-                            <a href="index.php?Page_Type=Site&amp;id=
-                                <?php echo $params['user']->getHomeSite()->getId()?>">
-                                <?php xecho($params['user']->getHomeSite()->getShortName()) ?>
-                            </a>
+                            <?php
+                            $homeId = $params['user']->getHomeSite()->getId();
```

Comment:
Does seem so from other bits in the file, but to check, is this the correct indentation for this kind of block?

Reply:
yep, that is to say, a `<?php` tag doesn't increase the level of indentation.